components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/internal/UserFunctionsServiceComponent.java

```diff
@@ -79,6 +82,8 @@ protected void activate(ComponentContext ctxt) {
                     new CheckSessionExistenceFunctionImpl());
             jsFunctionRegistry.register(JsFunctionRegistry.Subsystem.SEQUENCE_HANDLER, "getAssociatedLocalUser",
                     getAssociatedLocalUserFunctionImpl);
+            jsFunctionRegistry.register(JsFunctionRegistry.Subsystem.SEQUENCE_HANDLER, "setAssociationToLocalUser",
```

Comment:
the function name looks odd in javascript world. better think of different natural name.

Reply:
fixed in 97e9bbd7c06d24eca3cfde4e5b5428a54ede4f30